src/pages/User/components/PullRequests/ErrorText.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ErrorText = ({ errorMessage }) => (
+const ErrorText = ({ errorMessage, notAUserErrorMsg }) => (
```

Comment:
notAUserErrorMsg is not needed here because you're sending error messages in errorMessage.